tests/templates/test_subroutines/test_flip_sign.py

```diff
@@ -134,3 +134,21 @@ def test_wire_empty_error(self, n_status, n_wires):
         """Assert error raised when given empty wires"""
         with pytest.raises(ValueError, match="expected at least one wire representing the qubit "):
             qml.FlipSign(n_status, wires=n_wires)
+
+    @pytest.mark.jax
+    def test_jax_jit(self):
+        import jax
+
+        basis_state = [1, 0]
+
+        dev = qml.device("default.qubit", wires=2)
+
+        @jax.jit
+        @qml.qnode(dev)
+        def circuit():
+            for wire in list(range(2)):
+                qml.Hadamard(wires=wire)
+            qml.FlipSign(basis_state, wires=list(range(2)))
+            return qml.state()
+
+        circuit()
```

Comment:
If possible, check the solution (it would be enough  to check with the non-jitted version)